fio.spec

```diff
@@ -52,7 +56,7 @@ FIO devel
 %setup -q
 find . -type f > source_files.txt
 # fix hard-coded /usr/bin/bash
-sed -i -e "s|/usr/bin/bash|$(command -v bash)|g" tools/genfio
+sed -i -e "s|/bin/bash|$(command -v bash)|g" tools/genfio
```

Comment:
What is this really changing?  `bash` usually is in `/bin/`.

Reply:
Agree with you, I keep the sed as I am not familiar with the packaging script and do want to miss something.

- [ ] Remove useless sed

Reply:
> Agree with you, I keep the sed as I am not familiar with the packaging script and do want to miss something.
> 
> * [x]  Remove useless sed

Fixed with commit 8d6fdece37215a0f915a287d4d821a180f2620b4